### ISSUE TITLE
Nextcloud 14 Upgrade issue. 

### ISSUE THREAD

**Issue by Happyfeet01** [nextcloud/server#11202]
Hello, 

I made an Upgrade from nextcloud 13.06.1 to 14.0.0. After that I cant enter the settings. 
I get this message. 

```
The server was unable to complete the request .

If this occurs again, please send the following technical details to your server administrator.

Technische Details
Entfernte Adresse: 212.218.67.66
Anfragekennung: rYpWfHYesQE75gtzjiAx
```
Also this message 

```
The code integrity check has encountered problems. More information...
```
Nextcloud.log:

```
{"reqId":"xO7cp2NNoADOnHl94Kjt","level":3,"time":"2018-09-13T11:05:58+02:00","remoteAddr":"212.218.67.66","user":"Lars","app":"PHP","method":"GET","url":"\/index.php\/settings\/undefined\/img\/notifications.svg","message":"You are using a fallback implementation of the intl extension. Installing the native one is highly recommended instead. at \/var\/www\/cloud\/3rdparty\/patchwork\/utf8\/src\/Patchwork\/Utf8\/Bootup\/intl.php#18","userAgent":"Mozilla\/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit\/537.36 (KHTML, like Gecko) Chrome\/68.0.3440.106 Safari\/537.36","version":"14.0.0.19"}
{"reqId":"rVIEx6vrSGEe7xOoQaJX","level":3,"time":"2018-09-13T11:05:58+02:00","remoteAddr":"212.218.67.66","user":"Lars","app":"PHP","method":"GET","url":"\/ocs\/v2.php\/apps\/notifications\/api\/v2\/notifications","message":"You are using a fallback implementation of the intl extension. Installing the native one is highly recommended instead. at \/var\/www\/cloud\/3rdparty\/patchwork\/utf8\/src\/Patchwork\/Utf8\/Bootup\/intl.php#18","userAgent":"Mozilla\/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit\/537.36 (KHTML, like Gecko) Chrome\/68.0.3440.106 Safari\/537.36","version":"14.0.0.19"}
{"reqId":"P6bCSaFF7wIsix2ME4KH","level":3,"time":"2018-09-13T11:05:58+02:00","remoteAddr":"212.218.67.66","user":"Lars","app":"PHP","method":"GET","url":"\/index.php\/apps\/files\/","message":"You are using a fallback implementation of the intl extension. Installing the native one is highly recommended instead. at \/var\/www\/cloud\/3rdparty\/patchwork\/utf8\/src\/Patchwork\/Utf8\/Bootup\/intl.php#18","userAgent":"Mozilla\/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit\/537.36 (KHTML, like Gecko) Chrome\/68.0.3440.106 Safari\/537.36","version":"14.0.0.19"}
{"reqId":"NhkozBRHGVDpHAPpZN1O","level":3,"time":"2018-09-13T11:06:03+02:00","remoteAddr":"212.218.67.66","user":"Lars","app":"PHP","method":"GET","url":"\/ocs\/v2.php\/apps\/notifications\/api\/v2\/notifications","message":"You are using a fallback implementation of the intl extension. Installing the native one is highly recommended instead. at \/var\/www\/cloud\/3rdparty\/patchwork\/utf8\/src\/Patchwork\/Utf8\/Bootup\/intl.php#18","userAgent":"Mozilla\/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit\/537.36 (KHTML, like Gecko) Chrome\/68.0.3440.106 Safari\/537.36","version":"14.0.0.19"}
{"reqId":"jactCefOE00bF4U8Ra4Z","level":3,"time":"2018-09-13T11:06:07+02:00","remoteAddr":"212.218.67.66","user":"Lars","app":"PHP","method":"PUT","url":"\/index.php\/apps\/notes\/notes\/251996","message":"You are using a fallback implementation of the intl extension. Installing the native one is highly recommended instead. at \/var\/www\/cloud\/3rdparty\/patchwork\/utf8\/src\/Patchwork\/Utf8\/Bootup\/intl.php#18","userAgent":"Mozilla\/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit\/537.36 (KHTML, like Gecko) Chrome\/68.0.3440.106 Safari\/537.36","version":"14.0.0.19"}
{"reqId":"KpOD40bOspVGSClFRsd1","level":3,"time":"2018-09-13T11:06:09+02:00","remoteAddr":"212.218.67.66","user":"Lars","app":"PHP","method":"PUT","url":"\/index.php\/apps\/notes\/notes\/251996","message":"You are using a fallback implementation of the intl extension. Installing the native one is highly recommended instead. at \/var\/www\/cloud\/3rdparty\/patchwork\/utf8\/src\/Patchwork\/Utf8\/Bootup\/intl.php#18","userAgent":"Mozilla\/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit\/537.36 (KHTML, like Gecko) Chrome\/68.0.3440.106 Safari\/537.36","version":"14.0.0.19"}
{"reqId":"dkgUCkDrtapbIurWs1jH","level":3,"time":"2018-09-13T11:06:21+02:00","remoteAddr":"212.218.67.66","user":"Lars","app":"PHP","method":"PUT","url":"\/index.php\/apps\/notes\/notes\/251996","message":"You are using a fallback implementation of the intl extension. Installing the native one is highly recommended instead. at \/var\/www\/cloud\/3rdparty\/patchwork\/utf8\/src\/Patchwork\/Utf8\/Bootup\/intl.php#18","userAgent":"Mozilla\/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit\/537.36 (KHTML, like Gecko) Chrome\/68.0.3440.106 Safari\/537.36","version":"14.0.0.19"}
{"reqId":"0Rp2EZ5iQOCzOLpaU1aI","level":3,"time":"2018-09-13T11:06:23+02:00","remoteAddr":"212.218.67.66","user":"Lars","app":"PHP","method":"PUT","url":"\/index.php\/apps\/notes\/notes\/251996","message":"You are using a fallback implementation of the intl extension. Installing the native one is highly recommended instead. at \/var\/www\/cloud\/3rdparty\/patchwork\/utf8\/src\/Patchwork\/Utf8\/Bootup\/intl.php#18","userAgent":"Mozilla\/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit\/537.36 (KHTML, like Gecko) Chrome\/68.0.3440.106 Safari\/537.36","version":"14.0.0.19"}
{"reqId":"tEo88pDAJZJBELCJjbYJ","level":3,"time":"2018-09-13T11:06:34+02:00","remoteAddr":"212.218.67.66","user":"Lars","app":"PHP","method":"GET","url":"\/ocs\/v2.php\/apps\/notifications\/api\/v2\/notifications","message":"You are using a fallback implementation of the intl extension. Installing the native one is highly recommended instead. at \/var\/www\/cloud\/3rdparty\/patchwork\/utf8\/src\/Patchwork\/Utf8\/Bootup\/intl.php#18","userAgent":"Mozilla\/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit\/537.36 (KHTML, like Gecko) Chrome\/68.0.3440.106 Safari\/537.36","version":"14.0.0.19"}
{"reqId":"ChKwuyNq6W5BRDOBfj7y","level":3,"time":"2018-09-13T11:07:03+02:00","remoteAddr":"212.218.67.66","user":"Lars","app":"PHP","method":"GET","url":"\/ocs\/v2.php\/apps\/notifications\/api\/v2\/notifications","message":"You are using a fallback implementation of the intl extension. Installing the native one is highly recommended instead. at \/var\/www\/cloud\/3rdparty\/patchwork\/utf8\/src\/Patchwork\/Utf8\/Bootup\/intl.php#18","userAgent":"Mozilla\/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit\/537.36 (KHTML, like Gecko) Chrome\/68.0.3440.106 Safari\/537.36","version":"14.0.0.19"}
```

Nextcloud 14.0.0
Ubuntu 18.04
Apache2

**Comment by Happyfeet01** [comment]
After installing php-intl I get this error Message

```
time":"2018-09-13T11:53:30+02:00","remoteAddr":"212.218.67.66","user":"Lars","app":"index","method":"GET","url":"\/index.php\/settings\/user","message":{"Exception":"Error","Message":"Call to undefined function style()","Code":0,"Trace":[{"file":"\/var\/www\/cloud\/lib\/private\/legacy\/app.php","line":622,"function":"include"},{"file":"\/var\/www\/cloud\/lib\/private\/Settings\/Manager.php","line":365,"function":"getForms","class":"OC_App","type":"::","args":["personal"]},{"file":"\/var\/www\/cloud\/settings\/Controller\/CommonSettingsTrait.php","line":93,"function":"getPersonalSections","class":"OC\\Settings\\Manager","type":"->","args":[]},{"file":"\/var\/www\/cloud\/settings\/Controller\/CommonSettingsTrait.php","line":46,"function":"formatPersonalSections","class":"OC\\Settings\\Controller\\PersonalSettingsController","type":"->","args":["personal","personal-info"]},{"file":"\/var\/www\/cloud\/settings\/Controller\/CommonSettingsTrait.php","line":125,"function":"getNavigationParameters","class":"OC\\Settings\\Controller\\PersonalSettingsController","type":"->","args":["personal","personal-info"]},{"file":"\/var\/www\/cloud\/settings\/Controller\/PersonalSettingsController.php","line":57,"function":"getIndexResponse","class":"OC\\Settings\\Controller\\PersonalSettingsController","type":"->","args":["personal","personal-info"]},{"file":"\/var\/www\/cloud\/lib\/private\/AppFramework\/Http\/Dispatcher.php","line":166,"function":"index","class":"OC\\Settings\\Controller\\PersonalSettingsController","type":"->","args":["personal-info"]},{"file":"\/var\/www\/cloud\/lib\/private\/AppFramework\/Http\/Dispatcher.php","line":99,"function":"executeController","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->","args":[{"__class__":"OC\\Settings\\Controller\\PersonalSettingsController"},"index"]},{"file":"\/var\/www\/cloud\/lib\/private\/AppFramework\/App.php","line":118,"function":"dispatch","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->","args":[{"__class__":"OC\\Settings\\Controller\\PersonalSettingsController"},"index"]},{"file":"\/var\/www\/cloud\/lib\/private\/AppFramework\/Routing\/RouteActionHandler.php","line":47,"function":"main","class":"OC\\AppFramework\\App","type":"::","args":["OC\\Settings\\Controller\\PersonalSettingsController","index",{"__class__":"OC\\AppFramework\\DependencyInjection\\DIContainer"},{"section":"personal-info","_route":"settings.PersonalSettings.index"}]},{"function":"__invoke","class":"OC\\AppFramework\\Routing\\RouteActionHandler","type":"->","args":[{"section":"personal-info","_route":"settings.PersonalSettings.index"}]},{"file":"\/var\/www\/cloud\/lib\/private\/Route\/Router.php","line":297,"function":"call_user_func","args":[{"__class__":"OC\\AppFramework\\Routing\\RouteActionHandler"},{"section":"personal-info","_route":"settings.PersonalSettings.index"}]},{"file":"\/var\/www\/cloud\/lib\/base.php","line":989,"function":"match","class":"OC\\Route\\Router","type":"->","args":["\/settings\/user"]},{"file":"\/var\/www\/cloud\/index.php","line":42,"function":"handleRequest","class":"OC","type":"::","args":[]}],"File":"\/var\/www\/cloud\/apps\/ocdownloader\/settings\/personal.php","Line":17,"CustomMessage":"--"},"userAgent":"Mozilla\/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit\/537.36 (KHTML, like Gecko) Chrome\/68.0.3440.106 Safari\/537.36","version":"14.0.0.19"}
```

**Comment by macmeck** [comment]
same here on one of my machines

Nextcloud 14.0.0
Ubuntu 18.04
nginx

**Comment by Happyfeet01** [comment]
Maybe Ubuntu 18.04 related?

**Comment by macmeck** [comment]
I just figured out that for me it was an old app that crashed the settings pages... nextcloud.log identified issues with the old mozilla_sync app that still was enabled.

`sudo -u www-data php occ app:disable mozilla_sync`
did the trick for me.

In other cases different apps might be involved but checking nextcloud.log should give you an idea.

**Comment by Happyfeet01** [comment]
Okay. For me it was the App ocdownloader.

**Comment by Newyork311** [comment]
I resolved this error installing php 7.1 " php-intl " Nextcloud 14.0.3 On CentOs 6.9

**Comment by RandieM** [comment]
An older version of the Collabora Online app could also be causing this issue, see here: https://github.com/nextcloud/richdocuments/issues/297.
